src/App.js

```diff
@@ -1,11 +1,56 @@
 import Header from './Header';
 import Card from './Card';
+import { useState } from 'react';
 
 function App() {
+  const cards = [
+    {headerData: 'Caption 1', bodyData: 'text 1', id: 1},
+    {headerData: 'Caption 2', bodyData: 'text 2', id: 2},
+    {headerData: 'Caption 3', bodyData: 'text 3', id: 3},
+    {headerData: 'Caption 4', bodyData: 'text 4', id: 4},
+    {headerData: 'Caption 5', bodyData: 'text 5', id: 5},
+    {headerData: 'Caption 6', bodyData: 'text 6', id: 6},
+    {headerData: 'Caption 7', bodyData: 'text 7', id: 7},
+    {headerData: 'Caption 8', bodyData: 'text 8', id: 8}
+  ]
```

Comment:
Well, you should better add it to the App state. This will be the central storage of cards for your sample.
Like stored in the state, you should apply these changes to App state cards data once save icon of Card component is pressed.